### ISSUE TITLE
`colmena apply-local --sudo` succeeds but with exit code 101

### ISSUE THREAD

**Issue by bjornfor** [zhaofengli/colmena#111]
Everything seems to work just fine, but the exit code is 101:

```
$ colmena apply-local --sudo
[...]
bf-laptop ✅ 0s Pushed system closure
bf-laptop ✅ 2s Activation successful
$ echo $?
101
```

(I expect tools to return status 0 when successful.)

I'm using colmena from NixOS 22.05:
```
$ colmena --version
Colmena 0.3.0
```

**Comment by zhaofengli** [comment]
The privilege escalation method for `apply-all` has been changed in #85, so this specific symptom is no longer reproducible on `main`. The underlying issue has been fixed in 7d0a6eed90a2cf3d8152b3461c1ac2c7def424dd and 596fd9b12c3e060b5619493cda5716218f8c86c2 (0.3.x), and a new 0.3 release will be cut soon.

**Comment by bjornfor** [comment]
@zhaofengli: Thank you!
